cmake/Modules/Packaging.cmake

```diff
@@ -0,0 +1,44 @@
+set(CPACK_PACKAGE_NAME "valkey")
+
+valkey_parse_version(CPACK_PACKAGE_VERSION_MAJOR CPACK_PACKAGE_VERSION_MINOR CPACK_PACKAGE_VERSION_PATCH)
```

Comment:
can we consider a custom versioning scheme for the unstable branch such as `${CPACK_PACKAGE_NAME}-unstable-${CMAKE_SYSTEM_NAME}`? It will look like `${CPACK_PACKAGE_NAME}-255.255.255-${CMAKE_SYSTEM_NAME}` otherwise. I am a bit more in favor of explicitness. Separate PR is totally fine.

Reply:
+1 for `${CPACK_PACKAGE_NAME}-unstable-${CMAKE_SYSTEM_NAME}` for unstable branch